week6/hw6_ig.py

```diff
@@ -47,6 +47,6 @@ def map_like(fun, *elems):
 
```

Comment:
The function should work correctly with any type of iterable oblects:

example input: {'one': 1, 'two': 2}
actual output: {'one': 1, 'two': 2} 
expected: o n e t w o